drivers/ethernet/eth_sam_gmac.c

```diff
@@ -1807,14 +1804,56 @@ static void generate_mac(u8_t mac_addr[6])
 #endif
 }
 
+void monitor_work_handler(struct k_work *work)
```

Comment:
This function should be "static" as there are no requirements for it to be visible outside this compilation unit.

Reply:
Sure, I will fix that.